### ISSUE TITLE
Trying to Comment

### ISSUE THREAD

**Issue by dysabled** [instabot-py/instabot.py#1696]
Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [x ] I have searched for other issues with the same problem. (including #1153)

## Hosting
- [x ] Are you hosting the bot on a VPS or other Cloud hosting?
- [x ] If so; have you tried running the code on your local machine?

#### Select one:
- [ ] Bug
- [ ] Feature Request
- [x ] Technical Help
- [ ] Other

### Operating System? (include version)
- [ ] MAC
- [x ] Windows
- [ ] Linux
- [ ] Android

## Python Version?
- [ ] 2.x
- [x ] 3.x

## Description of your issue
Ive ran into a lot of issues and thought I had fixed them all until this came along, everytime i load up the bot this is all it does, im not comment blocked on ig & have tried running this on a vpn & still nothing. Someone please help.
```
Include the contents of the error.log here (if applicable). < fnaccz at 29.10.2018_20:08 fnaccz login success!
fnaccz at 29.10.2018_20:08 Get Media by tag: fortnitefortniteChristmas
fnaccz at 29.10.2018_20:08 Removing already liked medias..
fnaccz at 29.10.2018_20:08 Get Media by tag: fortnitely
fnaccz at 29.10.2018_20:08 Removing already liked medias..
fnaccz at 29.10.2018_20:08 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:08 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:08 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:08 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:08 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:08 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:08 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:09 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:09 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:09 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:09 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:09 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:09 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:09 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:09 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:09 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:09 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:09 Trying to comment: 1901177830881242903
fnaccz at 29.10.2018_20:09 Trying to comment: 1901177830881242903
```

## Config
3.71 python
windows 10
```
Include the contents of your config here (REMOVE YOUR USERNAME AND PASSWORD!) (if applicable). < REPLACE THIS TEXT
```

**Comment by DarkSuniuM** [comment]
Closed all the issues till 2019-01-09, We updated the repo, Please create a new issue if you need! Thanks
